internal/server/providers/git/git_e2e_test.go

```
@@ -0,0 +1,256 @@
+		t.Fatalf("write README: %v", err)
+	}
+	runGit(t, repo, "add", "README.md")
+	runGit(t, repo, "commit", "-m", "initial")
+	return repo
+}
+
+// runGit shells out to git in repo and fails the test on any non-zero
+// exit. Briefing explicitly allows shell-outs for fixture setup.
+func runGit(t *testing.T, repo string, args ...string) {
+	t.Helper()
+	cmd := exec.Command("git", args...)
```

golangci-lint (2.12.1)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)